Epsilon.Canvas/CanvasModuleCollectionFetcher.cs

```diff
@@ -21,26 +21,29 @@ IOutcomeHttpService outcomeService
         _outcomeService = outcomeService;
     }
 
-    public async IAsyncEnumerable<ModuleOutcomeResultCollection> GetAll(int courseId)
+    public async IAsyncEnumerable<ModuleOutcomeResultCollection> GetAll(int courseId, string[] allowedModules)
```

Comment:
Can you make sure that this conforms with the interface, so `allowedModules` -> `modules`